statsmodels/stats/_lilliefors.py

```diff
@@ -216,21 +308,37 @@ def kstest_normal(x, pvalmethod='approx'):
     '''
 
     x = np.asarray(x)
-    z = (x-x.mean())/x.std(ddof=1)
-    nobs = len(z)
+    nobs = len(x)
+
+    if dist == 'norm':
+        z = (x-x.mean())/x.std(ddof=1)
+        test_d = stats.norm.cdf
+        lilliefors_table = get_lilliefors_table(dist='norm')
+    elif dist == 'exp':
+        z = x
+        ex = stats.expon(scale=1./x.mean())
```

Comment:
Are you certain the tables are valid if we take that approach?

See the reference procedure in 

```
Lilliefors, H.W.
On the Kolmogorov-Smirnov test for the exponential distribution with mean unknown.
Journal of the American Statistical Association, Vol 64, No. 325. (1969), pp. 387–389.
```

which estimates the `lambda` parameter as the mean of `x`, rather than standardizing data as is typical for the normal case.